src/services/staff/staff-router.ts

```diff
@@ -0,0 +1,89 @@
+import { Router } from "express";
+import { StatusCodes } from "http-status-codes";
+import { StaffValidator } from "./staff-schema";
+import { Database } from "../../database";
+import RoleChecker from "../../middleware/role-checker";
+import { Role } from "../auth/auth-models";
+
+const staffRouter = Router();
+
+// GET /staff
```

Comment:
np: have comments explain what the endpoint is for in a sentence, like `Get all staff`